int32 agv_mode
bool agv_stopped
